stylesheets/_component.progress-bars.scss

```diff
@@ -1,41 +1,115 @@
 // Outer container
 .progress {
-  background-color: $progress-bg;
-  border-radius: $border-radius;
-  box-shadow: inset 0 1px 2px rgba(0, 0, 0, .1);
-  height: $line-height-computed;
-  margin-bottom: $line-height-computed;
-  overflow: hidden;
+    background-color: $progress-bg;
+    border-radius: $border-radius;
+    box-shadow: inset 0 1px 2px rgba(0, 0, 0, .1);
+    height: $line-height-computed;
+    margin-bottom: $line-height-computed;
+    overflow: hidden;
 }
 
 // Bar of progress
 .progress-bar {
-  background-color: color(primary);
-  box-shadow: inset 0 -1px 0 rgba(0, 0, 0, .15);
-  color: color(white);
-  float: left;
-  font-size: $font-size-small;
-  height: 100%;
-  overflow: hidden;
-  padding-right: 5px;
-  text-align: right;
-  text-shadow: 0 1px 0 rgba(0, 0, 0, .15);
-  transition: width .6s ease;
-  width: 0%;
+    background-color: color(primary);
+    box-shadow: inset 0 -1px 0 rgba(0, 0, 0, .15);
+    color: color(white);
+    float: left;
+    font-size: $font-size-small;
+    height: 100%;
+    overflow: hidden;
+    padding-right: 5px;
+    text-align: right;
+    text-shadow: 0 1px 0 rgba(0, 0, 0, .15);
+    transition: width .6s ease;
+    width: 0%;
 }
 
 .progress-bar--success {
-  background-color: color(success);
+    background-color: color(success);
 }
 
 .progress-bar--warning {
-  background-color: color(warning);
+    background-color: color(warning);
 }
 
 .progress-bar--danger {
-  background-color: color(danger);
+    background-color: color(danger);
 }
 
 .progress-bar--compact {
-  height: 5px;
+    height: 5px;
 }
+
+// Stepped Progressbar
+.stepped-progressbar {
+    display: table;
+    margin-bottom: 30px;
+    padding: 20px;
+    width: 100%;
+
+    .step {
+        display: block;
+        float: left;
+        overflow: hidden;
+        position: relative;
+        width: 50px;
+
+        &.last span::after {
+            width: 0;
+        }
+
+        &.successful span::after {
+            border-top: 2px dotted color(success);
+        }
+    }
+
+    .step-box {
+        background-color: color(white);
+        border: 1px solid color(gray);
+        border-radius: 50%;
+        display: block;
+        font-size: 16px;
+        height: 30px;
+        position: relative;
+        text-align: center;
+        width: 30px;
+        zoom: 1;
+
+        &.icon,
```

Comment:
I don't see an example of a `.step-box.icon` in the markup example. Is this line needed?

Reply:
Should really use `class="icon icon-flag-checkered"` in the examples, the `[class*="icon-"]` is a fallback selector for when the `.icon` parent class isn't present, it's not as performant I don't think...

Reply:
@nvasileiadis Where you have

```
.step-box {
    ... some styles ...
    &.icon,
    [class*='icon-'] {
        ... some styles...
    }
```

It doesn't look like you need the `&.icon`, as in your examples I don't see an element with both the `step-box` *and* the `icon` class. By using `&` it will compile to 

```
.step-box.icon,
.step-box [class*='icon-'] {
```

And it doesn't look like you need the first line, unless I'm missing something?